Cargo.toml

```diff
@@ -248,6 +248,8 @@ msaatext = []
 mscat = []
 mschapp = []
 mssip = []
+mswsock = []
+mswsockdef = []
```

Comment:
As the `mswsockdef` module is under `shared`, it should be listed under `# shared` in `Cargo.toml`